Site/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "veupathdb-react-scripts run-site-dev-server --config webpack.config.local.mjs --port 3000"
+    "start": "veupathdb-react-scripts run-site-dev-server --config webpack.config.local.mjs --port 3000",
+    "prepare": "relative-deps"
```

Comment:
How exciting! Is this safe to merge now, or will we need to give `EbrcWebsiteCommon` a `prepare` or `postinstall` script first?

Reply:
Oh my, I didn't really mean to include that, but maybe I should?

Reply:
It uses the `build` script of the dependency, so it works in this case.

Reply:
Perfect! I say merge away.

Reply:
This does have implications for website builds. I think we should wait until web-common is removed from project_home.

Reply:
Sounds like a plan

Reply:
I will remove this specific script, but leave the rest of the changes related to relative-deps.